docs/api/vast-parser.md

```diff
@@ -330,6 +332,27 @@ vastParser.parseVAST(vastXml, options)
   });
 ```
 
+### getEstimatedBitrate()
+
+Returns the average of the estimated bitrates in bits by secondes.
```

Comment:
```suggestion
Returns the average of the estimated bitrates in bytes by secondes.
```

Reply:
ok this comment is releveant only if we switched to byterate 😛

Reply:
As discussed via PM, we'll stick to the market unit which is kbps